dev-libs/granite/granite-5.2.4.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+VALA_MIN_API_VERSION=0.40
+
+inherit xdg-utils meson vala
```

Comment:
Just checked, xdg-utils doesnt seem to be exporting any functions, so this list can be sorted alphabetically without ebuild breaking?

Reply:
Yep, oversight! Thanks!